### PR TITLE
Create organization delete and update methods.

### PR DESCRIPTION
**Pull request by pasindur99**
Create organization delete and update methods.

### CODE REVIEW

File: src/main/java/com/services/OrganizationService.java [hunk 1]
```diff
@@ -84,7 +80,40 @@ public Organization saveOrganization(String name){
...
+            organization = getOne(id);
+
+        } catch (SQLException throwable) {
+            throwable.printStackTrace();
+        } catch (ClassNotFoundException e) {
+            e.printStackTrace();
+        }
+        return organization;
+    }
+}
```
Comment (jayasanka-sack) [inline src/main/java/com/services/OrganizationService.java:119]: EOL

File: src/main/java/com/OrganizationController.java [hunk 2]
```diff
@@ -69,39 +47,32 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
...
-        System.out.println("id - " + id);
-        Organization updatedOrganization = service.update(id,name);
+        response.setContentType("text/json");
         PrintWriter out = response.getWriter();
-        out.println(new Gson().toJson(updatedOrganization));
+        out.println(new Gson().toJson(organization));
     }
-     */
 
 }
```
Comment (jayasanka-sack) [inline src/main/java/com/OrganizationController.java:78]: EOL

File: src/main/java/com/OrganizationController.java [hunk 3]
```diff
@@ -69,39 +47,32 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
...
 
-    private int retrieveUserid(HttpServletRequest request) {
-        String pathInfo = request.getPathInfo();
-        if (pathInfo.startsWith("/")){
-            pathInfo = pathInfo.substring(1);
-        }
-        return Integer.parseInt((pathInfo));
+        response.setContentType("text/json");
+        PrintWriter out = response.getWriter();
+        out.println(new Gson().toJson(organizationService.getAll()));
```
Comment (jayasanka-sack) [inline src/main/java/com/OrganizationController.java:63]: No need to send a response when performing a delete

File: src/main/java/com/OrganizationController.java [hunk 4]
```diff
@@ -69,39 +47,32 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response) t
...
-
         }
     }
 
-/*
+    @Override
     protected void doDelete(HttpServletRequest request, HttpServletResponse response) throws ServletException,
             IOException {
 
         System.out.println("DELETE");
```
Comment (jayasanka-sack) [inline src/main/java/com/OrganizationController.java:57]: Remove all souts

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]
Well done. Please fix the requested minor changes

File: src/main/java/com/OrganizationController.java (hunk 4 above)
Comment (pasindur99) [reply src/main/java/com/OrganizationController.java:57]: Thank you @jayasanka-sack . A small thing I need to clarify.
Souts means these @Override symbols right?

Comment (jayasanka-sack) [reply src/main/java/com/OrganizationController.java:57]: nope. System.out.printlns :)

Comment (pasindur99) [reply src/main/java/com/OrganizationController.java:56]: Fixed with 2b5516c

Comment (pasindur99) [reply src/main/java/com/OrganizationController.java]: Done. Fix with https://github.com/pasindur99/Organizations-backend/pull/5/commits/072d8ed5489e0bd8dc51ed1a72db35cd2f08af6b

**Review by jayasanka-sack — APPROVED** [review]
Perfect!

**Pull request merged**